### PR TITLE
Add more content to the Readme.

### PR DESCRIPTION
**Pull request by siemensikkema**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,117 @@
...
+
+```swift
+.package(url: "https://github.com/vapor/multipart-kit.git", from: ...)
+```
+
+### Supported Platforms
+
+MultipartKit supports the following platforms:
+
+- Ubuntu 16.04+
```
Comment (0xTim) [inline README.md:48]: We dropped 16.04 support, this should be 18.04 or just link to the Swift docs as we support whatever they support

**Review by 0xTim — APPROVED** [review]
LGTM - just one comment about version support

File: README.md (hunk 1 above)
Comment (siemensikkema) [reply README.md:48]: @0xTim https://swift.org/platform-support/ still says 16.04 though 🤔

Comment (0xTim) [reply README.md:48]: I'm a couple of days early! 😅 

We announced it for Vapor back in May - https://forums.swift.org/t/vapor-dropping-support-for-ubuntu-16-04/49672

Ubuntu 16.04 is EOL so shouldn't be supported any more

Comment (siemensikkema) [reply README.md:48]: @0xTim I'm a bit lost as to how "a couple of days early" and an announcement in June (referring to May) about dropping support at the end of the month fit together 😅 Will Swift drop support once 5.5 lands?
Anyway, I'll change it to 18 for now and we can see if we can align on referring to Swift once that changes.

Comment (0xTim) [reply README.md]: Oops sorry I missed a link https://forums.swift.org/t/dropping-support-for-ubuntu-16-04/51726 - Swift will drop support today.
We dropped it in June, ahead of this alongside NIO and the rest of the SSWG (I got my dates mixed up 😆 )

**Comment by 0xTim** [comment]
This is good to merge I think

**Comment by 0xTim** [comment]
@siemensikkema is this ready to go?

**Pull request merged**